studio/src/fields/multifield.js

```diff
@@ -96,6 +96,14 @@ class MasMultifield extends LitElement {
     /* c8 ignore start */
     // Handle drag start
     dragStart(e, index) {
+        const activeElement = this.shadowRoot.activeElement;
+        if (
+            activeElement &&
+            activeElement.tagName.toLowerCase() === 'mas-mnemonic-field'
```

Comment:
1. Should I use a more generic approach, instead of targeting mas-mnemonic-field I can see if the tagName contains -field?
2. Yes, this doesn’t remove the drag functionality, it simply excludes focused textfield elements from it so that the user can select text inside (currently it’s not possible).